.vscodeignore

```diff
@@ -1,30 +1,20 @@
-.vscode/**
-.github/**
-.vscode-test/**
-out/**
-test/**
-src/**
-scripts/**
-**/*.map
-.gitignore
-tsconfig.json
-tsconfig.**.json
-biome.jsonc
-images/**
-vsc-extension-quickstart.md
-logo.svg
-node_modules
-package-lock.json
-bun.lockb
-yarn.lock
-bunfig.toml
-.devcontainer/**
-sheriff.config.ts
-build/**
-svgo.config.js
-.eslintignore
-material-colors.yml
-changelog.config.json
-dist/types/**
-dist/module.cjs
+# Files to include in the extension package
+# Documentation for this format:
+# https://code.visualstudio.com/api/working-with-extensions/publishing-extension#using-.vscodeignore
+!.npmignore
+!CHANGELOG.md
+!LICENSE.md
+!README.md
+!logo.png
 
+!package.json
+!package.nls.*.json
+
+!dist/extension
+!dist/module
```

Comment:
the module directory is only used for the npm package, so not needed here.

Reply:
Also Done